src/pages/nft/class/_classId.vue

```diff
@@ -154,7 +154,7 @@
                 'mb-[16px]',
                 'desktop:m-0',
               ]"
-              title="Owners"
+              title="Collectors"
```

Comment:
Perhaps add to translation file

Reply:
I just started to refactor this page, I will move the rest text to locales asap.